src/components/Book.jsx

```diff
@@ -17,26 +17,32 @@ const Book = ({ data }) => {
   };
```

Comment:
- [ ]  Hi, You have done a great job. I would advise you not to disable the linter because it is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs(source: [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software))).
- I added below the picture of the concerned file because you did not make the top side visible in this PR. 

![Capture](https://github.com/federicaulzurrun/Bookstore/assets/63972670/c2d467a7-c32d-4f2c-8ed3-4c7a39257ead)

Reply:
@OLIPLICHE Hii thanks!!! but that is not something for this specific project. is for the data on the book function if I delete that code is not going to work.

Reply:
@OLIPLICHE can you make it optional, please? I will work on it tomorrow during my free time. Because today is Saturday, I don't want to repeat the week for this. thank you so much!

Reply:
@federicaulzurrun. Alright, you can submit and the next reviewer gonna approve it. All the best